### ISSUE TITLE
[4,0] Currently Joomla 4.0 is a mess

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#15746]
at 9c4be6c92f6be7b30d1864a29ed31ebbf8333b10

On first loading the admin I get failed to initialize session

On first loading the frontend I get 

`Error displaying the error page: syntax error, unexpected '<<' (T_SL), expecting end of file: Error initialising the session.`

On loading the frontend again I get:

`Notice: Undefined property: JApplicationSite::$_language_filter in /Users/phil/Sites/joomla4/libraries/cms/application/site.php on line 471
`

After login to admin I get

`0 Argument 1 passed to JPlugin::setDispatcher() must implement interface Joomla\Event\DispatcherInterface, instance of JAuthentication given, called in /Users/phil/Sites/joomla4/libraries/cms/plugin/plugin.php on line 139`

Maybe @mbabker  ?

**Comment by PhilETaylor** [comment]
hmm might be a bad checkout... let me try again...

**Comment by PhilETaylor** [comment]
yup - bloody git corrupted - think my hard disk is about to fail, had issues all day :(

**Comment by mbabker** [comment]
Well if you have non-local issues to report, you know where to find us 😜

**Comment by PhilETaylor** [comment]
https://github.com/joomla/joomla-cms/pull/15706 ;-) can you fix that one ?
